packages/components/src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,193 @@
+    children,
+    disabled,
+    onClick,
+    value,
+}: AccordionItemProps) => {
+    return (
+        <RadixAccordion.Item
+            className={styles.accordionItem}
+            value={value}
+            onPointerDown={(event) => {
+                event.currentTarget.dataset.showFocusRing = 'false';
+            }}
+            onBlur={(event) => {
+                event.currentTarget.dataset.showFocusRing = 'true';
+            }}
+            onClick={onClick}
+            disabled={disabled}
+            data-test-id={dataTestId}
+        >
+            {children}
+        </RadixAccordion.Item>
+    );
+};
+AccordionItem.displayName = 'Accordion.Item';
+
+export type AccordionHeaderProps = {
+    /**
+     * Children of the Accordion header. This should contain `Accordion.Trigger`
+     */
+    children?: ReactNode;
+};
+
+export const AccordionHeader = ({ children }: AccordionHeaderProps) => {
+    return <RadixAccordion.Header>{children}</RadixAccordion.Header>;
+};
+AccordionHeader.displayName = 'Accordion.Header';
+
+export type AccordionTriggerProps = {
+    'data-test-id'?: string;
+    /**
+     * Children of the Accordion trigger. This contains the actually clickable and visible header content
+     */
+    children?: ReactNode;
+};
+
+export const AccordionTrigger = ({
+    'data-test-id': dataTestId = 'fondue-accordion-trigger',
+    children,
+}: AccordionTriggerProps) => {
+    return (
+        <RadixAccordion.Trigger className={styles.accordionTrigger} data-test-id={dataTestId}>
+            {children}
+            <IconCaretDown className={styles.accordionCaret} size="16" />
+        </RadixAccordion.Trigger>
+    );
+};
+AccordionTrigger.displayName = 'Accordion.Trigger';
+
+type AccordionContentProps = {
+    'data-test-id'?: string;
+    /**
+     * Children of the Accordion content. This contains the main content.
+     */
+    children?: ReactNode;
+    /**
+     * Adds a divider line between the header and the content.
+     */
+    divider?: boolean;
+    /**
+     * Click callback for the content.
+     */
+    onClick?: MouseEventHandler<HTMLDivElement>;
+    /**
+     * Controls if we show paddings around the content.
+     * @default 'large'
+     */
+    padding?: AccordionPadding;
+};
+
+export const AccordionContent = ({
+    'data-test-id': dataTestId = 'collapsible-wrap',
+    children,
+    divider = false,
+    onClick,
+    padding = 'large',
+}: AccordionContentProps) => {
+    return (
+        <RadixAccordion.Content
+            className={styles.accordionContent}
+            onClick={onClick}
+            data-test-id={dataTestId}
+            data-item-padding={padding}
+            data-item-divider={divider}
+        >
+            <div className={styles.accordionContentText} data-test-id={`inner-${dataTestId}`}>
+                {children}
+            </div>
+        </RadixAccordion.Content>
+    );
+};
+AccordionContent.displayName = 'Accordion.Content';
+
+export const Accordion = {
```

Comment:
Don't forget to re-export in the `index.ts` file too

Reply:
Ah right! 🤦‍♂️